doc/deployment_examples/sinfonier/feeding_sinfonier.md

```diff
@@ -0,0 +1,408 @@
+#<a name="top"></a>Sinfonier deployment of Cosmos
+Content:
+
+* [Introduction](#section1)
+* [Architecture](#section2)
+* [Orion contextBroker](#section3)
+* [Cygnus](#section4)
+    * [Configuration](#section4.1)
+    * [Running](#section4.2)
+* [Kafka](#section5)
+    * [Zookeeper](#section5.1)
+	   * [Configuration](#section5.1.1)
+	   * [Running](#section5.1.2)
+    * [Brokers](#section5.2)
+      * [Configuration](#section5.2.1)
+      * [Running](#section5.2.2)
+* [Sinfonier](#section6)
+* [General procedure step-by-step](#section7)
+* [Reporting issues and contact information](#section8)
+
+##<a name="section1"></a>Introduction
+This document describes how `Sinfonier` can consume stored context information from [Orion](http://catalogue.fiware.org/enablers/publishsubscribe-context-broker-orion-context-broker).
```

Comment:
can consume historic context information handled by [Orion](link...) and stored by [Cygnus](link...)